### PR TITLE
[Communication] Readme and changelog changes for March Release

### PR DESCRIPTION
**Pull request by ankitarorabit**
* Updated SMS to Beta.4
* Updated all other package versions to Beta.6

### CODE REVIEW

File: sdk/communication/azure-communication-sms/README.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ Azure Communication SMS is used to send simple text messages.
 <dependency>
   <groupId>com.azure</groupId>
   <artifactId>azure-communication-sms</artifactId>
-  <version>1.0.0-beta.4</version>
+  <version>1.0.0-beta.5</version>
```
Comment (minnieliu) [inline sdk/communication/azure-communication-sms/README.md:24]: Is this supposed to be beta.5 or 6? I see that in the pom.xml below, it is 6 but here, its 5

Comment (ankitarorabit) [reply sdk/communication/azure-communication-sms/README.md:24]: SMS and phonenumbers are going to be on beta.5 .. I think I should have added this in the notes

**Comment by RezaJooyandeh** [comment]
Let's use one version for all the packages of each language

**Review by RezaJooyandeh — APPROVED** [review]
:shipit:

File: sdk/communication/pom.xml [hunk 2]
```diff
@@ -24,17 +24,17 @@
...
           <groupId>com.azure</groupId>
           <artifactId>azure-communication-common</artifactId>
-          <version>1.0.0-beta.5</version> <!-- {x-version-update;com.azure:azure-communication-common;current} -->
+          <version>1.0.0-beta.6</version> <!-- {x-version-update;com.azure:azure-communication-common;current} -->
         </dependency>
         <dependency>
           <groupId>com.azure</groupId>
           <artifactId>azure-communication-identity</artifactId>
-          <version>1.0.0-beta.5</version> <!-- {x-version-update;com.azure:azure-communication-identity;current} -->
+          <version>1.0.0-beta.6</version> <!-- {x-version-update;com.azure:azure-communication-identity;current} -->
```
[the remark this answers is not in the document]
Comment (ankitarorabit) [reply sdk/communication/pom.xml:37]: They should, yes.. fixed it..

**Review by JianpingChen — APPROVED** [review]

**Pull request merged**